contrib/mac/app/README.md

```diff
@@ -0,0 +1,16 @@
+Julia OS X packaging
```

Comment:
I wonder if Apple provides official marketing guidelines on the OS X vs macOS naming issue.